autoscale/src/main/java/com/sequenceiq/periscope/config/ScalingActivityCleanupConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.periscope.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ScalingActivityCleanupConfig {
+
+    @Value("${periscope.scaling-activity.cleanup-duration.hours:24}")
```

Comment:
Nit: maybe rename to something like 'cleanup-event-age.hours'? Duration seems a little confusing.

Reply:
Updated